### PR TITLE
Add in time of day relating to most active periods for moderation

### LINKED ISSUE #71: Add in time of day relating to most active periods for moderation
> The most active 60, 30, 15, 10 and 5 minutes within 4 windows of time; midnight to 6am (early), 6am to 12pm (normal), 12pm -6pm (normal), 6pm -midnight (late) will be extracted from GGIR and used to test the effect of physical activity proximity to bedtime and wake time on sleep.

### PR DESCRIPTION
**Pull request by tarensanders**
Fixes #71

Waiting for the pipeline to finish then I'll mark as ready for review

### CODE REVIEW

File: R/clean_data.R [hunk 1]
```diff
@@ -100,7 +101,13 @@ clean_data <- function(data_joined, region_lookup) {
       sleep_onset_time = chron(times = sleep_onset_time),
       sleep_wakeup_time = chron(times = sleep_wakeup_time)
     ) %>%
-    select(-measurementday)
+    select(-measurementday)  %>% 
+    # Fix up the most active time
+    mutate(
+      pa_mostactivehr = 
+        hour(lubridate::ymd_hms(pa_mostactivehr)) +
```
Comment (conig) [inline R/clean_data.R]: `hour` and `minute` need lubridate attached or, we need lubridate added to requires for this function.

**Review by conig — CHANGES_REQUESTED** [review]
Check out requires in clean_data to see if require(lubridate) is needed.
Am I going nuts or is the model diagnostics table missing from multiverse.pdf despite being present in the rmd?

File: R/clean_data.R (hunk 1 above)
Comment (tarensanders) [reply R/clean_data.R]: Thanks, good catch.

Comment (tarensanders) [reply R/clean_data.R]: Thanks, good catch.

**Comment by tarensanders** [comment]
> Am I going nuts or is the model diagnostics table missing from multiverse.pdf despite being present in the rmd?

Ahh I didn't make a target for it. Interesting that it just fails quietly. Very good catch.

**Comment by tarensanders** [comment]
This should be good to go now

**Pull request merged**